.bamQC.wdl

```diff
@@ -0,0 +1,1130 @@
+version 1.0
```

Comment:
What does this file do? Could it have been committed by mistake?

Reply:
Thanks, my mistake, it somehow sneaked in.